kubeflow/openmpi/service.libsonnet

```diff
@@ -1,6 +1,8 @@
 {
   all(params):: [
     $.service(params),
+    $.serviceAccount(params),
+    $.clusterRoleBinding(params),
   ],
 
   name(params):: params.name,
```

Comment:
To simplify, can you delete this and replace `$.name(params)` with `params.name` throughout the file